README.rst

```diff
@@ -1192,6 +1194,12 @@ Just normal step declaration with the dependency injection should be used.
         pass
 
 
+Strict gherkin option is removed, so the strict_gherkin parameter can be removed from the scenario decorators
```

Comment:
We should apply standard formatting to code blocks like strict_gherkin, or pytst_bdd_step_validation_error

Reply:
i committed right from the web UI :)